src/win/snprintf.c

```diff
@@ -0,0 +1,49 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Yes.  FWIW, I'm the copyright holder of this particular piece of code and I wrote it long after I left Joyent.